marklogic-data-hub-central/ui/src/components/load/load-card.tsx

```diff
@@ -465,7 +465,7 @@ const LoadCard: React.FC<Props> = (props) => {
 
                   <div className={styles.cardNonLink} data-testid={`${elem.name}-toExistingFlow`}>
                                     Add step to an existing flow
-                    {selectVisible ? <HCTooltip id="missing-permission-tooltip" text={"Load: "+SecurityTooltips.missingPermission} placement={"bottom"}><div className={styles.cardLinkSelect}><div className={styles.cardLinkSelect}>
+                    {selectVisible ? <HCTooltip id="missing-permission-tooltip" show={!props.canWriteFlow} text={"Load: "+SecurityTooltips.missingPermission} placement={"bottom"}><div className={styles.cardLinkSelect}><div className={styles.cardLinkSelect}>
```

Comment:
Should we have a unit tests that covers this scenario?

Reply:
Yes, was added a new e2e for this scenario, after different probes, is more stable than UI tests.